runner/setup_github_actions_runner.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+sudo apt update
+sudo add-apt-repository ppa:deadsnakes/ppa -y
+sudo add-apt-repository ppa:git-core/ppa -y
+sudo apt install software-properties-common python3.7 python3.7-dev git -y
+
+mkdir ~/actions-runner 
+cd ~/actions-runner
+curl -O -L https://github.com/actions/runner/releases/download/v2.272.0/actions-runner-linux-x64-2.272.0.tar.gz
+tar xzf ./actions-runner-linux-x64-2.272.0.tar.gz
+rm actions-runner-linux-x64-2.272.0.tar.gz
```

Comment:
You'd need to update in 4 places when you update the version. Maybe just keep the version and url as variable and just do `curl -Ls ${URL} | tar ...`?

Reply:
Btw, is it possible to pass version from terraform to the script?

Reply:
@skshetry sure, we can just add arg to bash script, just as we do with Github token. Will do that.

Reply:
@skshetry added github actions runner version as a parameter. In the future, we can think of automating this and just obtaining the latest version with some script.